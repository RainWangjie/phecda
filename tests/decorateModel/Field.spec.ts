import { Field } from '../../src'

describe('Model Field', () => {
  it('should create instance', () => {
    const field = new Field({
      displayName: 'test',
    })

    expect(JSON.stringify(field)).toMatchSnapshot()
  })
})
