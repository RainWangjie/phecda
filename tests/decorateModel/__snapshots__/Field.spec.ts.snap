// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Model Field should create instance 1`] = `"{\\"displayName\\":\\"test\\"}"`;
